src/mui/list/Datagrid.js

```diff
@@ -4,7 +4,7 @@ import FlatButton from 'material-ui/FlatButton';
 import ContentSort from 'material-ui/svg-icons/content/sort';
 import title from '../../util/title';
 
-const styles = {
+const defaultStyles = {
```

Comment:
@fzaninotto 
We should add more documentation about what is expected when overriding the Datagrid styles. Should we address this in another PR so that we can merge this one now ?